schema/collections.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import type { GraphQLFieldConfig } from "graphql"
+import { pageable } from "relay-cursor-paging"
+import { connectionFromArraySlice } from "graphql-relay"
+import _ from "lodash"
+import gravity from "../lib/loaders/gravity"
+import cached from "./fields/cached"
+import { artworkConnection } from "./artwork"
+import filterArtworks, { filterArtworksArgs } from "./filter_artworks"
+import { queriedForFieldsOtherThanBlacklisted, parseRelayOptions } from "../lib/helpers"
+import { GravityIDFields, NodeInterface } from "./object_identification"
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLBoolean } from "graphql"
+
+const CollectionType = new GraphQLObjectType({
+  name: "Collection",
+  interfaces: [NodeInterface],
+  isTypeOf: obj => _.has(obj, "name") && _.has(obj, "private") && _.has(obj, "default"),
+  fields: {
+    ...GravityIDFields,
+    cached,
+    artworks_connection: {
+      type: artworkConnection,
+      args: pageable({}),
+      resolve: ({ id }, options, request, { rootValue: { accessToken, userID } }) => {
+        const gravityOptions = parseRelayOptions(options)
+        gravityOptions.user_id = userID
+        gravityOptions.total_count = true
+
+        // TODO: what do to with this?
+        delete gravityOptions.page
+
+        return gravity
+          .with(accessToken, { headers: true })(`collections/artworks`, gravityOptions)
```

Comment:
Some of those are pretty old, and I think maybe a Folio album or two? Either way, you should be good to just use `collection/saved-artwork/artworks` (and hardcode that), a la https://github.com/artsy/gravity/blob/48c54d8b4aa6ec3a8ee5bbdfa123dbee7c02d474/app/models/domain/collection.rb#L56